controllers/topology/internal/mergepatch/mergepatch_test.go

```diff
@@ -820,19 +820,149 @@ func Test_enforcePath(t *testing.T) {
 		},
 		{
 			name: "Ignore partial match",
-			simpleMap: map[string]interface{}{
+			authoritativeMap: map[string]interface{}{
 				"foo": "a",
 			},
 			twoWaysMap: map[string]interface{}{},
 			path:       contract.Path([]string{"foo", "bar", "baz"}),
 			want:       map[string]interface{}{},
 		},
+
+		{
+			name: "authoritative has no changes, twoWays has not changes, no changes",
```

Comment:
```suggestion
			name: "authoritative has no changes, twoWays has no changes, no changes",
```
nit, type (same below)